### ISSUE TITLE
class io.intercom.android.sdk.gcm.GcmBroadcastReceiver does not exist

### ISSUE THREAD

**Issue by muazzam27** [intercom/intercom-android#270]
class io.intercom.android.sdk.gcm.GcmBroadcastReceiver does not exist

Getting the Above Error on the App start on Android Studio 2.2.

**Comment by harkin** [comment]
hi @muazzam27 

Could you give me some more details like what versions of Intercom you're using in your app and the play services library present too? It seems like you're using a version from the 1.x line which is pretty old, I'd encourage you to update to 3.x where we have a more up to date GCM integration.

**Comment by muazzam27** [comment]
hi @harkin Thanks for your swift reply

yes i am using intercom version from 1.x and that is 
```
compile 'io.intercom.android:intercom-sdk-base:1.1.11@aar'
compile 'io.intercom.android:intercom-sdk-gcm:1.1.11@aar'

```
and GCM version ` compile 'com.google.android.gms:play-services-gcm:8.4.0'`

but When i update the latest intercom version which is this compile 'io.intercom.android:intercom-sdk:3.+'

i got more, some of the methods are not available in this Version like i am able to access these method that are available beofore

```
Intercom.client().setupGCM(token, R.drawable.intercomsdk_default_push);
Intercom.client().openGCMMessage(getIntent());
```

Above methods are not available in the latest version of Intercome.
in short got more issues when i update Intercome.

**Comment by ronocod** [comment]
Is multidex being used in this app? It may be causing this issue if not correctly set up.

I'd strongly recommend updating to version 3 of the Messenger too, you can find a guide [here](https://docs.intercom.com/a-guide-to-the-new-intercom-messenger/upgrade-to-the-new-messenger-android) on how to migrate your existing code over. You only need a couple of changes :)

**Comment by ronocod** [comment]
Closing this as it's an integration issue, not an issue with the library itself.
